src/shared/lib/ILazyMobXTableApplicationDataStore.ts

```diff
@@ -149,7 +150,7 @@ export class SimpleGetterLazyMobXTableApplicationDataStore<T>
     }
 
     @computed get showingAllData() {
-        return this.tableData.length === this.allData.length;
+        return _.flatten(this.tableData).length === this.allData.length;
```

Comment:
why is all data ungrouped?  it should be the number of rows that table can possibly display (agnostic to what is being displayed a row (a group of things or one thing or an array of one, whatever).